.travis.yml

```diff
@@ -4,7 +4,10 @@ services:
 - docker
 language: go
 go:
-  - "1.11.x"
+  - "1.13.x"
```

Comment:
Please bump the version of go in `.go-version` to match so releases and accetance tests on TeamCity remain consistent

Reply:
This didn't need to be bumped, so I've reverted it.